### DISCUSSION TITLE
Help with background color mismatches in Artisan 2.6

### DISCUSSION THREAD

**Discussion by 210doc** [artisan-roaster-scope/artisan#938] [Q&A] [answered]
I am in the process of preparing to roast coffee on a new Huky. I have installed the Phidget RTD probes and a VINT hub and installed Artisan. In the configure mode for colors, my button colors are spot on, but when trying to set the on screen background graph colors to match the traditional green, yellow/orange, brown gradients, the colors are not even in the same hue. I tried all the different choices of "classic" etc with no remedy. Artisan is loaded on an older HP 64bit laptop running Win10 and an Nvidia  graphics card. I have no color issues with it when using lightroom etc.
I find no reference in any help file or Michael's guide trying different color spaces. It's not a big deal....unless you're an anal retentive coffee geek. TIA for any direction.
Mike

**Comment by MAKOMO** [comment]
I guess some screenshots would help to understand the issue. You can just drag-and-drop them on a comment field directly here on GitHub.

**Reply by 210doc** [reply]
I am new as of today to github, so will try to attach. Thx for the quick reply

**Comment by 210doc** [comment]
![image](https://user-images.githubusercontent.com/111084530/184235023-afaa1e7d-1e4d-4ebc-92f4-301e1d667148.png)

**Comment by 210doc** [comment]
![image](https://user-images.githubusercontent.com/111084530/184235092-6ca7c98f-8b31-47d0-86d1-c08f1efd59d1.png)

**Comment by 210doc** [comment]
![image](https://user-images.githubusercontent.com/111084530/184235331-8b72d929-9ec0-496f-969c-932dec957129.png)
I am copying screenshot from the clipboard and pasting here. I can preview and it looks good, but when I click "answer", it appears nothing is sent??

**Reply by 210doc** [reply]
here it is with the background scheme selected
![image](https://user-images.githubusercontent.com/111084530/184236113-6727b8ac-a3fe-4644-8ffa-b67fdf518a91.png)

**Comment by 210doc** [comment]
![image](https://user-images.githubusercontent.com/111084530/184236778-ad2fb9f2-5119-436b-bdb3-e171ffdeb252.png)

**Comment by 210doc** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/111084530/184236903-98f26e63-c54e-44aa-b98d-895f2c5213c1.png)
If this image is still the wrong one (in spite of preview showing correctly) I'll try to find another way to take a photo with my phone and email. I don';t want to fill up this thread with dozens of the incorrect screenshot posting. I am unable to drag and can only select the snippit and copy/paste which obviously isn't working well.

**Reply by 210doc** [reply]
weird...this is my color space selections finally

**Reply by roasterdave** [reply]
Sorry, I do not follow.  What is the problem?   Nothing appears out of place.  Can you explain what is and is not showing as expected in these screenshots?

**Reply by 210doc** [reply]
I am new to artisan and trying to configure a somewhat traditional presentation. If you note my graph is striped at the bottom during dry phase, even though configured for a shade of green per the artisan palatte, ramp phase is showing pink and is configured for yelllow, development is configured for brown and is a lavender hue on my screen. 
The configuration palette for buttons is spot on per my selection. Thanks for responding

**Reply by roasterdave** [reply]
Those horizontal bands to indicate the phases in the background of the graph are rendered with transparency (they are not opaque).  The transparency naturally changes the color's appearance.  The colors are properly rendered in these screenshots.  

Note that buttons are rendered with zero transparency (fully opaque).

>  If you note my graph is striped at the bottom during dry phase, even though configured for a shade of green per the artisan palatte

The horizontal band for drying phase does not show because your **Drying min** temperature is the same as the **Drying max** temperature.  Go to **Config>> Phases** and set **Drying min** to a lower temperature, like 212 degF or 100 degC.  The light grey stripes at the bottom are indicators for special [events](https://artisan-scope.org/docs/events/) like your Gas setting or Air Flow.  

![image](https://user-images.githubusercontent.com/30009312/184259325-94dd3420-b0d7-432c-8aca-36cb9478aa1b.png)

>ramp phase is showing pink and is configured for yelllow

According to your screen shot the color of the Maillard Phase (ramp) is #ffaa00 which is a shade of orange.  The horizontal band is rendering that color with transparency and looks correct from the screenshot.

>development is configured for brown and is a lavender hue on my screen

Agreed that the Finishing Phase is brown, #996633.  On my screen your screenshot shows a band that is a light brown as expected.  I do not see any tint of lavender.

Everything in your screenshots looks fine from here and as expected.  Adjust the **Phase min** temperature and you should be good to go.

**Reply by poundy** [reply]
if you're new to Artisan, why not just use the preset themes? That way you can get on with life and not worry about this stuff, and tweak it to your liking later.  I still use Classic today.

**Reply by 210doc** [reply]
@roasterdave  Thanks for that suggestion. I’ll make that correction to the dry phase lower T° tomorrow. Ironically I did my first roast today, and the colors displayed during the roast were close to being the default colors phases I was trying to achieve. I wasn’t getting a MET curve today, but after reading an answer Michael gave someone on HB Re a missing curve with a center device being caused by the wrong label in the extra device configuration, I reconfigured my extra from RTD A to RTD B and problem solved.

@poundy  “Get on with liife?”…that’s always helpful technical advice. Thx. Possibly you might have missed the part where I said I  had tried  Classic? ……as that is exactly what I want, just like Dave’s screenshot above. Before posting I spent undue time trying classic and every other choice after the classic default was so off on my screen.  I’m appreciative that RoasterDave has been kind enough to provide actual useful help with a newb’s Artisan issues.

**Reply by 210doc** [reply]
For the benefit of others new to Artisan and just a FYI if you’re repurposing an older laptop. My older laptop with an Nvidia graphics card failed to accurately display my chosen colors during setup, however they are spot on “classic” as desired when actually roasting.
 RDaves suggestion to edit temp ranges also expanded the green phase space and still leaves room for fan and burner markers as well as the ET ROR being out of the way in the lower area.
I’ve looked, but unable to find a way to designate my     issue “answered”, but it has been, and once again my thx.
